src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,109 @@
+import cN from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+import { Loader } from '../Loader';
+
+interface TabsProps {
+  people: Person[];
+  isLoading: boolean;
+  hasError: boolean;
+}
+
+import { useParams } from 'react-router-dom';
+
+export const PeopleTable: React.FC<TabsProps> = ({
+  people,
+  isLoading,
+  hasError,
+}) => {
+  const { selectedUserSlug } = useParams();
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {hasError && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
```

Comment:
Consider adding more specific error handling or logging to provide more information about what went wrong.